src/coreComponents/python/modules/geosx_xml_tools_package/geosx_xml_tools/unit_manager.py

```diff
@@ -46,54 +56,94 @@ def buildUnits(self):
-                    'psf':        {'value': 1853.184,       'alt': [],                      'usePrefix': True}}
+        imp_defs = {
+            "pound": {
+                "value": 0.453592,
+                "alt": ["lb", "pounds", "lbs"],
+                "usePrefix": True,
+            },
+            "poundforce": {"value": 0.453592 * 9.81, "alt": ["lbf"], "usePrefix": True},
+            "stone": {"value": 6.35029, "alt": ["st"], "usePrefix": True},
+            "ton": {"value": 907.185, "alt": ["tons"], "usePrefix": True},
+            "inch": {
+                "value": 1.0 / (3.281 * 12),
+                "alt": ["in", "inches"],
+                "usePrefix": False,
+            },
+            "foot": {"value": 1.0 / 3.281, "alt": ["ft", "feet"], "usePrefix": True},
+            "yard": {"value": 3.0 / 3.281, "alt": ["yd", "yards"], "usePrefix": True},
+            "rod": {"value": 16.5 / 3.281, "alt": ["rd", "rods"], "usePrefix": True},
+            "mile": {
+                "value": 5280.0 / 3.281,
+                "alt": ["mi", "miles"],
+                "usePrefix": True,
+            },
+            "acre": {"value": 4046.86, "alt": ["acres"], "usePrefix": True},
+            "gallon": {
+                "value": 0.00378541,
+                "alt": ["gal", "gallons"],
+                "usePrefix": True,
+            },
+            "psi": {"value": 6894.76, "alt": [], "usePrefix": True},
+            "psf": {"value": 1853.184, "alt": [], "usePrefix": True},
+        }
 
         # Other commonly used units:
-        other_defs = {'dyne':       {'value': 1.0e-5,    'alt': ['dynes'],              'usePrefix': True},
-                      'bar':        {'value': 1.0e5,     'alt': ['bars'],               'usePrefix': True},
-                      'atmosphere': {'value': 101325.0,  'alt': ['atm', 'atmospheres'], 'usePrefix': True},
-                      'poise':      {'value': 0.1,       'alt': ['P'],                  'usePrefix': True},
-                      'barrel':     {'value': 0.1589873, 'alt': ['bbl', 'barrels'],     'usePrefix': True},
-                      'horsepower': {'value': 745.7,     'alt': ['hp', 'horsepowers'],  'usePrefix': True}}
+        other_defs = {
+            "dyne": {"value": 1.0e-5, "alt": ["dynes"], "usePrefix": True},
+            "bar": {"value": 1.0e5, "alt": ["bars"], "usePrefix": True},
+            "atmosphere": {
+                "value": 101325.0,
+                "alt": ["atm", "atmospheres"],
+                "usePrefix": True,
+            },
+            "poise": {"value": 0.1, "alt": ["P"], "usePrefix": True},
+            "barrel": {
+                "value": 0.1589873,
+                "alt": ["bbl", "barrels"],
+                "usePrefix": True,
+            },
+            "horsepower": {
+                "value": 745.7,
+                "alt": ["hp", "horsepowers"],
+                "usePrefix": True,
+            },
+        }
```

Comment:
This is the kind of issues I do not like with autoformatter.
Sometimes you want to impose specific formatting because it's more readable...
Whatever 🤷